lib/model/folder_sendrecv.go

```diff
@@ -1976,7 +1976,7 @@ func (f *sendReceiveFolder) checkToBeDeleted(file, cur protocol.FileInfo, hasCur
 		return err
 	}
 	if fs.IsNotExist(err) {
-		if hasCur && !cur.Deleted {
+		if hasCur && !cur.Deleted && !cur.IsUnsupported() {
```

Comment:
I wasn't sure whether this should be the generic test for unsupported flag or if it should be `runtime.GOOS == windows && cur.IsSymlink()`. Currently we only use the unsupported flag for this use case, so it applies. And if we keep using that flag for stuff we do *not* ever write to disk (which makes sense when it is unsupported), the generic check is sensible.